makesignum.c

```diff
@@ -79,14 +82,10 @@ int main(void)
         wprintf(L"#define MAXSIGIDX %zu\n\n", v);
     }
 
-    wprintf(L"/* number of realtime signals that can be handled by yash */\n");
-    wprintf(L"#define RTSIZE %d\n\n",
 #if defined SIGRTMIN && defined SIGRTMAX
-            SIGRTMAX - SIGRTMIN + 10
-#else
-            0
+    wprintf(L"/* number of realtime signals that can be handled by yash */\n");
+    wprintf(L"#define RTSIZE %d\n\n", SIGRTMAX - SIGRTMIN + 10);
```

Comment:
Could you revert these lines? `RTSIZE` should always be defined, even if all current uses of `RTSIZE` are guarded by `#if defined SIGRTMIN && defined SIGRTMAX`.
There's a chance that future me will forget that RTSIZE is defined conditionally, which would cause a subtle bug where compilation fails only in certain environments.